### PR TITLE
Moved locale configuration from runtime properties to properties file.

### PR DESCRIPTION
**Pull request by eea02**
Locale configuration should be moved to properties file instead current configuration in runtime properties - database.

### CODE REVIEW

**Comment by skodape** [comment]
Some explanation-motivation would be nice.

Also it looks like we are missing default for older version of configuration file - will this raise version to 3.0.0?

**Comment by eea02** [comment]
During implementation of internationalization support for UV, decision was made to configure language in installation/configuration time and later it cannot be changed without restart of application. This behaviour suits more for properties file than runtime properties, that can be changed during runtime.

This change is a legacy issue, because at the implementation time, locale was already set in runtime properties.

About default value: as far as I know this issue is still open. It is still not decided if newer version of application should be able to use out-dated configuration file.

**Comment by skodape** [comment]
Default value: ok

The localization can be changed without frontend restart, just update property and reload page (F5). Or at least this work when I tried last time on local host - so this is not desired functionality, or it does not work properly?

Thank, for response.

**Comment by eea02** [comment]
Yes, frontend language can be changed in runtime, but this adds problem to backend. Imagine that you change the language in runtime, refresh the application and try to run a pipeline. Backend will communicate with you in former language. You will end up with mixed languages, which is not desirable. To avoid such problems, we should enforce the language setting, only during install/config phase.

**Comment by skodape** [comment]
Now I see, many thanks for explanation.

**Comment by skrchnavy** [comment]
@tomas-knap to comment

File: backend/conf/config.sample.properties [hunk 1]
```diff
@@ -75,3 +75,6 @@ email.admin = admin@example.com
 # Configuration of DPU modules
 module.path = {full path to "odcs" (home) dir of the project}/target
 
+# Language configuration (e.g.: en_US, sk)
+locale = sk
```
Comment (tomas-knap) [inline backend/conf/config.sample.properties]: Default should be EN

File: frontend/src/main/webapp/WEB-INF/config.sample.properties [hunk 2]
```diff
@@ -95,3 +95,10 @@ module.path = {full path to "odcs" (home) dir of the project}/target
 #header.color1 = #5f5247
 #header.color2 = #5f5247
 #header.color3 = #5f5247
+
+# credentials for Master REST API
+master.api.user=master
+master.api.password=commander
+
+# Language configuration (e.g.: en_US, sk)
+locale = sk
```
Comment (tomas-knap) [inline frontend/src/main/webapp/WEB-INF/config.sample.properties]: Default should be "en"

**Comment by tomas-knap** [comment]
Please set "en" as default. Otherwise Ok.

**Pull request merged**